src/main/java/fi/okm/jod/yksilo/entity/YksilonSuosikki.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2024 The Finnish Ministry of Education and Culture, The Finnish
+ * The Ministry of Economic Affairs and Employment, The Finnish National Agency of
+ * Education (Opetushallitus) and The Finnish Development and Administration centre
+ * for ELY Centres and TE Offices (KEHA).
+ *
+ * Licensed under the EUPL-1.2-or-later.
+ */
+
+package fi.okm.jod.yksilo.entity;
+
+import fi.okm.jod.yksilo.domain.SuosikkiTyyppi;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Check;
+
+@Entity
+@Getter
+@Builder
+@Table(indexes = {@Index(columnList = "yksilo_id")})
+// TODO: move constraint to migration tool and add koulutusmahdollisuus type when available
+@Check(constraints = "tyomahdollisuus_id IS NOT NULL AND tyyppi = 'TYOMAHDOLLISUUS'")
+@AllArgsConstructor
+@NoArgsConstructor
+public class YksilonSuosikki {
+  @GeneratedValue @Id private UUID id;
+
+  @Column(name = "luotu", nullable = false, updatable = false)
+  private LocalDateTime createdAt;
```

Comment:
Should be "Instant", localdatetime is "timestamp without time zone"

Reply:
Changed to Instant.